src/hooks/useNotificationTypes.tsx

```diff
@@ -212,6 +216,55 @@ export const notificationTypes: NotificationTypeConfig[] = [
       };
     },
   },
+  {
+    type: NotificationType.FunkitDeposit,
+    useTrigger: ({ trigger }) => {
+      const stringGetter = useStringGetter();
+      const funkitDeposits = useAppSelector(getFunkitDeposits, shallowEqual);
+      const { openAccountModal } = useAccountModal();
+
+      useEffect(() => {
+        // eslint-disable-next-line no-restricted-syntax
+        for (const deposit of Object.values(funkitDeposits)) {
+          const { checkoutId, status } = deposit;
+          trigger(
+            checkoutId,
+            {
+              icon: <Icon iconName={IconName.FunkitInstant} tw="text-color-accent" />,
+              title:
+                status === 'COMPLETED' || !ErrorStatuses.includes(status ?? '')
```

Comment:
ErrorStatuses can be enum.

Reply:
It was a bit challenging to get the types right, I think I can do this in a follow up